.travis.yml

```diff
@@ -88,6 +88,8 @@ install:
   - travis_retry sudo luarocks install busted
   # Install luacheck for "make check-qa".
   - if [ "$DO_CHECKQA" = 1 ]; then travis_retry sudo luarocks install luacheck; fi
+  # Install depgraph for "make check-requires".
+  - travis_retry sudo luarocks install depgraph
```

Comment:
Yes, and `check-qa` could then be moved to `LUA=5.1 LUANAME=lua5.1 BUILD_IN_DIR=/tmp/awesome-build` instead of the coverage build (the second slowest)